include/aspect/initial_temperature/adiabatic_boundary.h

```diff
@@ -33,29 +34,38 @@ namespace aspect
 
     /**
      * A class that computes an initial temperature field based
-     * on a user-defined adiabatic boundary for the 3D ellipsoid
-     * chunk geometry model. It discretizes the model domain into
+     * on a user-defined adiabatic boundary.It discretizes the model domain into
      * two regions separated by an isotherm boundary below which
      * the temperature increases adiabatically. Above the
      * user-defined isotherm boundary the temperature linearly
      * increases from a surface temperature (273.15 K or 0 degree C)
-     * to the isotherm (1673.15 K or 1600 degree C). The user
+     * to the isotherm (1673.15 K or 1400 degree C). The user
      * defines the location of the isotherm boundary with an ascii
-     * data file with the format defined in the ASPECT manual. Note
-     * that the latitudinal and longitudinal bounds of the ascii input
-     * data file need to be at least 1 degree wider than the bounds
-     * you use to define the region of your ellipsoid chunk geometry.
+     * data file with the format defined in the ASPECT manual.
      *
-     * This plugin is developed by Tahiry Rajaonarison and D. Sarah Stamps.
+     * This plugin is developed by Tahiry Rajaonarison, Emmanuel Njinju, and D. Sarah Stamps.
      */
     template <int dim>
-    class AdiabaticBoundary : public Interface<dim>, public SimulatorAccess<dim>
+    class AdiabaticBoundary : public Interface<dim>,  public Utilities::AsciiDataBoundary<dim>
     {
       public:
+
+        /**
+         * Constructor.
+         */
+        AdiabaticBoundary ();
+
+        void
+        initialize ();
+
+        // avoid -Woverloaded-virtual:
+        using Utilities::AsciiDataBoundary<dim>::initialize;
+
         /**
          * Return the initial temperature as a function of position.
          */
-        double initial_temperature (const Point<dim> &position) const override;
+        virtual
+        double initial_temperature (const Point<dim> &position) const;
```

Comment:
Please remove the `virtual` and put an `override` at the end of the line. The reason is explained here: #3258.